picard/ui/metadatabox/__init__.py

```diff
@@ -265,41 +270,104 @@ def keyPressEvent(self, event):
         else:
             super().keyPressEvent(event)
 
+    def get_selected_tags(self, items):
+        result = TagDiff()
+        for item in items:
+            tag, value = self.get_row_info(item.row())
+            col = item.column()
+            result.add(
+                tag=tag,
+                old=value[self.COLUMN_OLD] if col == self.COLUMN_OLD else None,
+                new=value[self.COLUMN_NEW] if col == self.COLUMN_NEW else None,
+                removable=self.tag_diff.status[tag] != TagStatus.NOTREMOVABLE,
+                readonly=self.tag_diff.status[tag] == TagStatus.READONLY
+            )
+
+        result.update_tag_names()
+        return result
+
+    def get_row_info(self, row):
+        tag = self.tag_diff.tag_names[row]
+        value = {}
+        value[self.COLUMN_OLD] = self.tag_diff.old[tag]
+        value[self.COLUMN_NEW] = self.tag_diff.new[tag]
+
+        return tag, value
+
     def _copy_value(self):
-        item = self.currentItem()
-        if item:
-            column = item.column()
-            tag = self.tag_diff.tag_names[item.row()]
-            value = None
-            if column == self.COLUMN_ORIG:
-                value = self.tag_diff.old[tag]
-            elif column == self.COLUMN_NEW:
-                value = self.tag_diff.new[tag]
+        if len(self.objects) > 1:
```

Comment:
Is this the right check to see if there are multiple files selected? @zas is reporting that this check isn't working correctly they only have 1 file selected, I'm unable to reproduce.

Reply:
Clearly not, because it contains 2 objects (a Track and a File) since it was tagged already:

```
D: 10:26:22,572 ui/metadatabox._copy_value:298: {<FLACFile '01 Ummon.flac'>, <Track 38851005-c0f5-4a59-a623-3589ae86db11 'Ummon'>}
D: 10:26:22,572 ui/metadatabox._copy_value:301: Unable to copy when multiple files are selected, clearing the clipboard
```

Reply:
Oh, OK, I'll go and do some reverse engineering to see what the best approach is